tests/test_helpers.rs

```diff
@@ -43,7 +43,9 @@ impl BrokerImpl {
 #[derive(Debug)]
 pub struct TestConfig {
     pub bootstrap_brokers: Vec<String>,
+    #[allow(dead_code)]
```

Comment:
not actually dead code, just not used by all end2end tests and I didn't wanna add yet another top-level crate just for the test infra.